doc/install.md

```diff
@@ -1,15 +1,35 @@
-### Building
+## Building and Installing Stacker
 
-Stacker requires at least go 1.11, since it uses go modules.
-On Ubuntu 18.04 you can get that using the instructions at:
+### Go Dependency
+
+Stacker requires at least go 1.11, since it uses Go modules.
+
+#### Ubuntu 18.04
+
+On Ubuntu 18.04 you can install Go using the instructions at:
 https://github.com/golang/go/wiki/Ubuntu
 
-Or you could get the latest version available at:
+#### Fedora 31
```

Comment:
I dont' really think that it is of much use to tell people how to get golang. Our instructions are out of date, and incomplete. I'd just give a link to golang wiki and say figure it out.

Reply:
Once again, carryover from the original docs; I don't really have much of a preference.  If @tych0 wants it gone, I'll be happy to drop it.